content/posts/android-tv-12.md

```diff
@@ -0,0 +1,143 @@
+
+### New player interface
+
+It's fun to see those design changes but the frequently used part of the app is the video player. The player design was completely rewritten to remove clutter and make it easier to use.
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-player.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-player.png" />
+</div>
+
+The new player interface is just the beginning. A project to rewrite the playback code is already in the works. This rewrite should help with the crashes and unnecessary transcoding that happens sometimes. We did however fix some issues. Notably are changes for Fire TV to direct play more often.
+
+## Fixed that bug
+
+Together with all the visible changes we've made tons of fixes to the code. We're now using [Kotlin] as our primary language, migrated completely to [AndroidX] and made a brand new [SDK] that is slowly being integrated for a more secure and stable connetion with your server.
+
+A complete list of all the fixed bugs can be found in our changelog linked below. It's a big list!
+
+[Kotlin]: https://kotlinlang.org
+[AndroidX]: https://developer.android.com/jetpack/androidx/
+[SDK]: https://github.com/jellyfin/jellyfin-sdk-kotlin
+
+## Going forward
+
+It took some time to get this release out. We didn't feel like the state of the app was good enough for a release for a while.
+Fortunately we worked hard to make sure we did feel confident and here we are! Starting from this release we're changing some things
+to make sure the next release won't take this long. Our brand new SDK that is already used in our Android app and third party apps is one of the tools helping us with faster releases. By having a more type-safe base we can prevent tons of crashes, allowing us to focus on actual features and bugs.
+
+We're already working on some new stuff like the earlier mentioned rewrite of the playback code. We're using feature flags for this so we can work on this code while still being able to publish new versions. Other changes that we're looking into are more improvements to the user interface, better Live TV, better music support and more bugfixing!
+
+## Contributors
+
+Like all releases, we couldn't do it without our contributors and your donations! Jellyfin is made entirely by volunteers that don't get paid. This release was made possible by the following contributors:
+
+**Jellyfin Team**
+
+- [@nielsvanvelzen](https://github.com/nielsvanvelzen) - [GitHub sponsors](https://github.com/sponsors/nielsvanvelzen)
+- [@thornbill](https://github.com/thornbill) - [GitHub sponsors](https://github.com/sponsors/thornbill)
+- [@MrChip53](https://github.com/MrChip53)
+- [@AndreasGB](https://github.com/AndreasGB)
+
+**Other contributors**
+
+- [@linetrimmer](https://github.com/linetrimmer)
+- [@Froghut](https://github.com/Froghut)
+- [@jellyfin-bot](https://github.com/jellyfin-bot)
```

Comment:
Should we include the bot as a contributor? 😄

Reply:
Nope, I did remove dependabot and github-actions[bot] but forgot to remove the jellyfin-bot.